### ISSUE TITLE
Synergy with THL Omaolo platform

### ISSUE THREAD

**Issue by obruck** [futurice/symptomradar#131]
THL is gathering location and time data of people with respiratory tract symptoms with Omaolo.fi. I would suggest finding synergy between these different platforms to reduce redundant double work.

**Comment by jnissin** [comment]
Agreed, I think we should look for ways of combining the data in a meaningful fashion. I think the first step is to look at the overlap between the questions in Omaolo and Oiretutka and take a look if we could somehow combine the two datasets at least to some extent. However, I don't think the Omaolo data is publicly accessible to everyone?

**Comment by T20v** [comment]
hi, 
here some quick and dirty ideas.  There is also  https://korona.kans.io/  that might give some synergy ideas how to publish data/co-operate.  My comments are  here  just as end user.  I feel like this would be like voting machine  to show categorized  single and group values calculated into generic area healthiness  index value per post number area.  That would be Corona heat map.  Get computing power from Veikkaus AB or from   äänestyskone  machine.   Idea is to have self service testing site with self service analysis to be able to show  reality data based on how individuals feel.     

- Data collection based on given  post number  (get the data from known government source)
- visualization to end user based on post number  (filter by post number or bigger entity group)
- mandatory to have opportunity to update my  given "points",  IF / WHEN people get sick later !
- questionare topics: 
- male/female
- post number as per drop down selection value  
- blood group:   known value given, not known (default)  , drop down ....
- ratings from 0 (none = ok) to 5 (severe heavy impact) to following
- temperature/ fever rating (drop down group categories)  35...36, 36...37, 37...38, 38...39, 39+
- head ace (päänsärky)
- sore throat (kurkkukipu)
- coffing   (yskä)
- sneezing (nuha)
- itching (kutina)
- skin changes  (ihomuutokset)
- breathing capability  (hengityskyky)
- chest pain (keuhkorintakivut)
- muscles or body pain (lihaskipu / yleiskipu)
- stomac pain  (vatsakipu)
- diarrhea  (ripuli)
- constipation  (ummetus)
- Corona identification,   NO, tested negative, I feel I have it, tested corona positive
- capability to move ( normal, only inside house, mainly only in bed, 
- capability to eat / dring
- treatment types: not sick, home care, (I was) in hospital, (I was) in intensive care
- those only from symptoms view
- many other but thats it

Take some ideas from the list in case found useful.

From here you get reference post number data  and  dat collection completion rate% can be measured
http://www.stat.fi/tup/paavo/index_en.html

Main thing is to get a simple heat map per post number/ aggregation of heat map to bigger area entities like Helsinki, Espoo, Vantaa, Uusimaa...


Br, 
Mr. T. 
DevOps trainee
